components/navigation/NavBar.js

```diff
@@ -127,7 +168,20 @@ export default function NavBar({
               <IconLoupe />
             </SearchButton>
 
-            <GithubButton text="Star on GitHub" href="https://github.com/asyncapi/spec" className="py-2 ml-2" inNav="true" />
+            {/* // Language Picker Component */}
+            {/* <LanguageSelect
+              options={uniqueLangs}
+              onChange={(value) => {
+                setLang(value.toLowerCase());
+                changeLanguage(value.toLowerCase());
+              }}
+              className=""
+              selected={i18n.language.toLocaleUpperCase()}
+            /> */}
```

Comment:
Are we going to use these lines of code in later part of PRs? If No, kindly remove this.

Reply:
Yes, that part will be un-commented in the last PR.